Adyen/UI/Form/Items/Address/AddressViewModel.swift

```diff
@@ -1,13 +1,13 @@
 //
-// Copyright (c) 2021 Adyen N.V.
+// Copyright (c) 2022 Adyen N.V.
 //
 // This file is open source and available under the MIT license. See the LICENSE file for more info.
 //
 
 import Foundation
 import UIKit
 
-/// :nodoc:
+@_spi(AdyenInternal)
 public enum AddressField: String, CaseIterable {
```

Comment:
maybe add comments to cases from [API Explorer](https://docs.adyen.com/api-explorer/#/CheckoutService/v68/post/payments__reqParam_billingAddress) ?